config/initializers/capybara.rb

```diff
@@ -0,0 +1,6 @@
+include Capybara::DSL

```

Comment:
Yup, I've just read around in stackoverflow, and found out about this too. got this fixed this in the next PR.